src/nupic/utils/LogItem.cpp

```diff
@@ -66,7 +66,7 @@ LogItem::~LogItem()
   }
 
   if (ostream_ == nullptr)
-    ostream_ = &(std::cout);
+    ostream_ = &(std::cerr);
```

Comment:
@scottpurdy, regarding
> This seems out of place in this PR. I don't have a good sense for what the right thing to do here is.

This is something I observed while debugging, and it seemed a trivial enought change versus going through the process in a separate PR. But I also see your point about this not being strictly necessary for the task at hand. I could move `ostream_ = &(std::cerr);` to a separate PR, if I must.

It's your call, please advise.